align/pnr/checker.py

```diff
@@ -52,6 +53,8 @@ def check_placement(placement_verilog_d, scale_factor):
                         llx=bbox.llx/scale_factor,
                         lly=bbox.lly/scale_factor,
                         urx=bbox.urx/scale_factor,
-                        ury=bbox.ury/scale_factor
+                        ury=bbox.ury/scale_factor,
+                        sx=1,  # TODO: change back to sX
```

Comment:
Not working right now for mirror constraint.

Reply:
@soneryaldiz This is the only failing problem in this branch right now

Reply:
This was a temporary check to force failure. I will revert now.